pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -358,7 +377,7 @@ func makeDeployment(opts ...deploymentOption) *appsv1.Deployment {
 }
 
 func revision(name, ns string, opts ...revisionOption) *v1.Revision {
-	revision := defaultRevision.DeepCopy()
+	revision := defaultRevision().DeepCopy()
```

Comment:
```suggestion
	revision := defaultRevision()
```
No need to deep copy anymore